src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,48 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
 public class SalaryInfo {
+    private static final String FORMAT_STRING = "dd.MM.yyyy";
```

Comment:
```suggestion
    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

lets make DateTimeFormatter as constant not only String format